sdk/core/azure-core/pom.xml

```diff
@@ -65,6 +65,11 @@
       <artifactId>jackson-dataformat-xml</artifactId>
       <version>2.10.1</version> <!-- {x-version-update;com.fasterxml.jackson.dataformat:jackson-dataformat-xml;external_dependency} -->
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.module</groupId>
+      <artifactId>jackson-module-afterburner</artifactId>
+      <version>2.10.1</version> <!-- {x-version-update;com.fasterxml.jackson.module:jackson-module-afterburner;external_dependency} -->
+    </dependency>
```

Comment:
I thought we decided to not use afterburner for now because we didn't see any perf wins. We should remove this now, correct?

Reply:
Hi Jonathan, cosmos team is also using after-burner. and we have seen perf gain.
I think you would see per gain for more complex Pojo where many reflection calls will be made in the absence of after-burner.